src/test/resources/data/typicalDataBundle.json

```diff
@@ -149,6 +149,7 @@
       "courseId": "idOfTypicalCourse1",
       "name": "Instructor1 Course1",
       "email": "instructor1@course1.tmt",
+      "key": "instructor1@course1.tmt%idOfTypicalCourse1739021015",
```

Comment:
Will use a comparison method to bypass. Why's specifying the key in the JSON discouraged?